src/modules/Dropdown/DropdownItem.js

```diff
@@ -75,7 +75,10 @@ DropdownItem.propTypes = {
   selected: PropTypes.bool,
 
   /** Display text. */
-  text: PropTypes.string,
+  text: PropTypes.oneOfType([
```

Comment:
While we're updating these, would you mind updating the `text` and `children` to being exclusive.  `customPropTypes` lives in `lib`:

```js
  /** Primary content. */
  children: customPropTypes.every([
    PropTypes.node,
    customPropTypes.disallow(['text']),
  ]),

  /** Display text. */
  text: customPropTypes.every([
    PropTypes.string,
    customPropTypes.disallow(['children']),
  ]),
```